sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/_utils.py

```diff
@@ -256,6 +256,12 @@ def _populate_part_a_fields(resource: Resource):
     return tags
 
 
+def _is_synthetic_source(properties: Attributes) -> bool:
+    # TODO: Use semconv symbol when released in upstream
+    synthetic_type = properties.get("user_agent.synthetic.type")  # type: ignore
```

Comment:
Assume this is just a hardcoded string here to avoid using an experimental Semantic Attribute?

Reply:
See the TODO. Upstream has not released the sem conv yet. Will update this with the symbol when they do.